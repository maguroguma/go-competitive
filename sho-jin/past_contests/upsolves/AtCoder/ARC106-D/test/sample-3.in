2 5
1234 5678
