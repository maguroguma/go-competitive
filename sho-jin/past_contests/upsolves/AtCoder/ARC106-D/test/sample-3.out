6912
47775744
805306038
64822328
838460992
